.github/workflows/main.yml

```
@@ -0,0 +1,30 @@
+name: build
+on:
+  # run on push but only for the main branch
+  push:
+    branches:
+      - master
```

Should this be `main`?
```suggestion
      - main
```

---

We use master as release branch
Should we move from `master` to `main` branch?

---

Ideally we would use `main` going forward. We recently migrated all the krakenjs repos and almost all the other paypal repos to use `main` instead of `master`.

---

Yeah I figure lets do this PR with `master` as the default branch and I'll open a subsequent PR to move to `main` with steps for what the team can do locally. That way its more easily linked and followed. I will do that immediately after merging this